### PR TITLE
assign bending angle as bit value

### PR DESCRIPTION
**Pull request by dildick**

### CODE REVIEW

File: DataFormats/CSCDigi/interface/CSCCorrelatedLCTDigi.h [hunk 1]
```diff
@@ -107,6 +113,7 @@ class CSCCorrelatedLCTDigi
   uint16_t syncErr;
   uint16_t cscID;
   float gemDPhi;
+  uint16_t gemDphi;
```
Comment (slava77) [inline DataFormats/CSCDigi/interface/CSCCorrelatedLCTDigi.h:116]: Sven

Please change the name of the new data member and accessor/modifier functions.
This one is a really terrible choice.
I suggest gemDPhiBits or smth of the sorts

Thanks

**Comment by dildick** [comment]
You're right. I'll change it.

File: L1Trigger/CSCTriggerPrimitives/src/CSCMotherboardME11.cc [hunk 2]
```diff
@@ -825,14 +826,28 @@ void CSCMotherboardME11::matchGEMPads(const GEMCSCPadDigiCollection* gemPads)
...
+
             //gem_bx = id_pad.second->bx();
           }
         }
         if (gem_matched)
         {
           if (debug_gem_matching) std::cout<<" GOT MATCHED GEM!"<<std::endl;
-          //lct.setGEMBX(gem_bx);
           lct.setGEMDPhi(min_dphi);
+	  // assing the bit value
```
Comment (slava77) [inline L1Trigger/CSCTriggerPrimitives/src/CSCMotherboardME11.cc:837]: Would it make sense to define the integer delta as just the distance in the trigger pads from a given LCT key-half-strip?
This is a somewhat unique definition.
The one based on 98% pt values is a bit complex and may need redefinitions if we choose different pt thresholds

**Comment by dildick** [comment]
Are the GE1/1 strips and ME1/1 halfstrips completely radial? If not, how do we solve this?

**Comment by slava77** [comment]
Sven, is your comment related to my comment on dphi binning or is it unrelated?

The GE1/1 strips are in general not radial (after we moved to v5 they are actually essentially radial, judging from the dphi(rec,simHit) plots shown in validation).
There is (or should be) and additional lookup table to correct for these.

**Pull request merged**